modules/farm_fd2_school/src/entrypoints/Vue1/App.vue

```diff
@@ -136,7 +113,33 @@
   </div>
 </template>
 
-<script></script>
+<script>
+export default {
+  data() {
+    return {
+      selected: 4,
+      unit: 1,
+      items: [
+        { id: 1, name: 'ARUGULA' },
```

Comment:
Because each crop is unique it is possible to simplify this by using an array of strings instead of an array of objects.  Similarly for the units below.